ui/components/multichain/notification-detail-collection/notification-detail-collection.tsx

```diff
@@ -30,11 +34,21 @@ export type NotificationDetailCollectionProps = {
   collection: string;
 };
 
+const isIpfsURL = (url: string): boolean => {
+  return url.startsWith('ipfs://');
+};
+
 export const NotificationDetailCollection: FC<
   NotificationDetailCollectionProps
 > = ({ icon, label, collection }) => {
   const { badgeSrc, src } = icon;
 
+  const isIpfsEnabled = useSelector(getIpfsGateway);
+  const openSeaEnabled = useSelector(getOpenSeaEnabled);
+
+  const ipfsImageIsRenderable = isIpfsEnabled && isIpfsURL(src) && src;
+  const openseaImageIsRenderable = openSeaEnabled && src && !isIpfsURL(src);
```

Comment:
NOTE - for other devs viewing. The NFT notifications we will show and push to users are ones that get vetted through the same API checks to prevent suspicious or malicious images that is also used in Portfolio.